### ISSUE TITLE
My map is a day ahead 

### ISSUE THREAD

**Issue by Tucker1077** [jeanropke/RDR2CollectorsMap#1281]
#### Found a new location? [Use this form here](https://github.com/jeanropke/RDR2CollectorsMap/issues/182)
* What day cycle it is?
* Coordinates or a clear image of the location
* Include the item name

Coordinates can be found by setting the `Show coordinates on click` to `Yes`

**Comment by OraleWey** [comment]
yeah mines too, pls someone fix it, check out my PC date time and check the date on the collector's map.
![Captura de pantalla 2024-12-21 204718](https://github.com/user-attachments/assets/68cfc201-262d-4a6c-b092-f001c7fbbc94)

**Comment by gebederry** [comment]
@OraleWey That is the date of the current item cycles, which updates daily at 00:00 UTC. When hovering over it with your mouse, please check the help info bar at the bottom.
